pkg/store/disk.go

```diff
@@ -57,32 +58,33 @@ func InspectDisk(diskName string) (*Disk, error) {
 	return disk, nil
 }
 
-// inspectDiskSize attempts to inspect the disk size by itself,
-// and falls back to inspectDiskSizeWithQemuImg on an error.
-func inspectDiskSize(fName string) (int64, error) {
+// inspectDiskSizeAndFormat attempts to inspect the disk size and format by itself,
+// and falls back to inspectDiskSizeAndFormatWithQemuImg on an error.
```

Comment:
Not WithQemuImg

Reply:
Do you mean to use `WithImgUtil` instead of `WithQemuImg`?

Reply:
Sorry, I was misreading the comment line